packages/core/app/utils/rsql-utils.js

```diff
@@ -0,0 +1 @@
+export { default } from 'navi-core/utils/rsql-utils';
```

Comment:
Isnt this just for mirage, should it be part of core? Can we just make it a mirage util?

Reply:
if so it should live under `addon/mirage/utils`?